pkg/bpf/map.go

```diff
@@ -405,3 +400,30 @@ func (m *Map) DeleteAll() error {
 
 	return nil
 }
+
+// openIfNotOpened opens Map m if it isn't opened.
+func (m *Map) openIfNotOpened() error {
+	if m.fd == 0 {
+		if err := m.Open(); err != nil {
```

Comment:
You're right - I don't think we really need this function anymore. I'll refactor accordingly.